templates/embedded/form-data.json

```diff
@@ -0,0 +1,289 @@
+              "label": "Other",
+              "value": "Other",
+              "hasTextarea": true
+            }
+          ]
+        },
+        {
+          "title": "Which products are you most interested in?",
+          "inputType": "checkbox",
+          "isRequired": true,
+          "fields": [
+            {
+              "type": "checkbox",
+              "label": "SMART START",
+              "value": "SMART START"
+            },
+            {
+              "type": "checkbox",
+              "label": "App Store",
+              "value": "App Store"
+            },
+            {
+              "type": "checkbox",
+              "label": "Board Enablement",
+              "value": "Board Enablement"
+            },
+            {
+              "type": "checkbox",
+              "label": "Maintenance",
+              "value": "Maintenance"
+            },
+            {
+              "type": "checkbox",
+              "label": "Expanded Security Maintenance (ESM)",
+              "value": "Expanded Security Maintenance (ESM)"
+            },
+            {
+              "type": "checkbox",
+              "label": "Technical Support",
+              "value": "Technical Support"
+            },
+            {
+              "type": "checkbox",
+              "label": "Training",
+              "value": "Training"
+            },
+            {
+              "type": "checkbox",
+              "label": "All of the above",
+              "value": "All of the above"
+            }
+          ]
+        },
+        {
+          "title": "Anything else you would like to tell us?",
+          "id": "any-other-comments",
+          "isRequired": false,
+          "fields": [
+            {
+              "type": "long-text",
+              "id": "any-other-comments"
+            }
+          ]
+        },
+        {
+          "title": "How should we get in touch?",
+          "id": "about-you",
+          "noCommentsFromLead": true,
+          "fields": [
+            {
+              "type": "text",
+              "id": "firstName",
+              "label": "First name",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "id": "lastName",
+              "label": "Last name",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "id": "company",
+              "label": "Company name",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "id": "title",
+              "label": "Job title",
+              "isRequired": true
+            },
+            {
+              "type": "email",
+              "id": "email",
+              "label": "Work email",
+              "isRequired": true
+            },
+            {
+              "type": "tel",
+              "id": "phone",
+              "label": "Mobile/cell phone number"
+            }
```

Comment:
Missing 'country' here, as on the [live form](https://ubuntu.com/embedded#get-in-touch)